### DISCUSSION TITLE
How to turn up the sensitivity on diarization?

### DISCUSSION THREAD

**Discussion by mcantillon21** [deepgram/community#558] [General help]
I need the diarization to be overly detective and sensitive. How can I make this happen?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@mcantillon21 

>  need the diarization to be overly detective and sensitive. How can I make this happen?

Could you give us a bit more details about what you are trying to do with Diarization?
